vsintegration/src/FSharp.ProjectSystem.Base/Project/ProjectReferenceNode.cs

```diff
@@ -805,6 +805,12 @@ private static FrameworkCompatibility CheckFrameworksCompatibility(ProjectNode t
                 return FrameworkCompatibility.Ok;
             }
 
+            if (String.Compare(otherFrameworkName.Identifier, ".NETStandard", StringComparison.OrdinalIgnoreCase) == 0)
+            {
+                // we always allow references to projects that are targeted to the ".NETStandard" family
```

Comment:
What happens if  versions are not compatible as per [this compatibility table](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support)?

e.g.

.NET Framework 4.5 project --> referencing --> .NET Standard 2.0 project

There are also differences depending on the version of the .NET Core SDK being used.

Reply:
After talking with @TIHan, it turns out that you _can_ add references to incompatible projects, but you can do that with C# too.  YOLO.

Reply:
This is ok. C# projects allow you to add netstandard references that would be incompatible, but it wont compile. F# projects will also not compile, so it's fine.